engine/src/test/java/com/datatorrent/stram/client/AppPackageTest.java

```diff
@@ -105,9 +106,30 @@ public void testAppPackage() throws Exception
 
     Assert.assertEquals("mydtapp-1.0-SNAPSHOT.jar", apps.get("MyFirstApplication").getString("file"));
 
+    String errorStackTrace = apps.get("MyFirstApplication").getString("errorStackTrace");
```

Comment:
Why is it necessary to rely on `apps`? @sandeshh Why was it necessary to introduce the Map/HashMap?